dev/TreeView/InteractionTests/TreeViewTests.cs

```diff
@@ -2896,6 +2896,168 @@ public void SingleSelectWithUnrealizedChildrenDoesNotMoveSelection()
+                root.Expand();
+                Wait.ForIdle();
+
+                ClickButton("DisableClickToExpand");
+                ClickButton("LabelItems");
+
+                Log.Comment("Select item \"Root.0\" before dragging it");
+                UIObject dragUIObject = VerifyAndSelectDragAndDropItem();
+
+                Log.Comment("Drag and drop selected item \"Root.0\" between \"Root.1\" and \"Root.2\"");
+                var height = dragUIObject.BoundingRectangle.Height;
+                // 5% from the edge of the next item 
+                var distance = (int)(height * 1.45);
+
+                Log.Comment("Starting Drag...distance:" + distance);
+                InputHelper.DragDistance(dragUIObject, distance, Direction.South);
+
+                TestEnvironment.VerifyAreEqualWithRetry(
+                    5,
+                    () => "Root | Root.1 | Root.0 | Root.2",
+                    () =>
+                    {
+                        ClickButton("GetChildrenOrder");
+                        return ReadResult();
+                    });
+
+                Log.Comment("Verify item \"Root.0\" is still selected after drop.");
+
+                ClickButton("GetSelected");
+                Verify.AreEqual("Selected: Root.0", ReadResult());
+
+                UIObject VerifyAndSelectDragAndDropItem()
+                {
+                    UIObject root0 = FindElement.ById("Root.0");
+                    Verify.IsNotNull(root0, "Verifying Root.0 is found");
+
+                    Log.Comment("Select item \"Root.0\" before dragging it");
+                    InputHelper.Tap(root0);
+
+                    ClickButton("GetSelected");
+                    Verify.AreEqual("Selected: Root.0", ReadResult());
+
+                    return root0;
+                }
+            }
+        }
+
+        [TestMethod]
+        [TestProperty("TestSuite", "B")]
+        public void VerifyTreeViewMultiSelectDragAndDropSelectedNodeDoesNotUnselect_NodeMode()
+        {
+            VerifyTreeViewMultiSelectDragAndDropSelectedNodeDoesNotUnselect();
+        }
+
+        [TestMethod]
+        [TestProperty("TestSuite", "B")]
+        public void VerifyTreeViewMultiSelectDragAndDropSelectedNodeDoesNotUnselect_ContentMode()
+        {
+            VerifyTreeViewMultiSelectDragAndDropSelectedNodeDoesNotUnselect(isContentMode: true);
+        }
+
+        private void VerifyTreeViewMultiSelectDragAndDropSelectedNodeDoesNotUnselect(bool isContentMode = false)
+        {
+            if (PlatformConfiguration.IsOSVersionLessThan(OSVersion.Redstone5))
+            {
+                // TODO 19727004: Re-enable this on versions below RS5 after fixing the bug where mouse click-and-drag doesn't work.
+                Log.Warning("This test relies on touch input, the injection of which is only supported in RS5 and up. Test is disabled.");
+                return;
+            }
+
+            if (isContentMode && IsLowerThanRS5())
```

Comment:
>IsLowerThanRS5 [](start = 33, length = 14)

Is this needed given that the test is disabled lower than RS5

Reply:
I copied it from existing tests. I asked myself the same question and I found it reasonable to be kept here because the reason why the test is already disabled above is due to a bug - with the comemnt even saying re-enable it once the bug is fixed. So assuming the bug is fixed, we will then need this check, otherwise the test will run in ContentMode on versions < RS5.

That was my original thinking here....but I just noticed...do we need this check anyway (regardless of a previous check disabling it or not)? We are testing WinUI APIs here, after all, so the TreeView ContentMode APIs (data binding) are available on all Windows versions we can run WinUI on. So...it looks to me this is a relict of the time when the TreeView tests ran using OS XAML instead of WInUI.